CHANGELOG

```diff
@@ -1,4 +1,14 @@
 # Changelog
+## [0.2.2] - 2019-02-03
+### New Features
+ - Team interface support
+
+### Bug Fixes
+ - Fix memory leak of NM.Client
+ - Get SR-IOV VFs configuration from runtime
+ - Fix DHCPv4 with static IPv6
+ - Learn the OVS Bridge ports and interfaces from NetworkManager
```

Comment:
I am not sure what this really means and I guess users will not know, too.

Reply:
Maybe that is something not important to users, it just means that now we are using libnm functions to get all bridge ports..